src/main/java/com/health/bmi_checker/bmiCheckerApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.health.bmi_checker;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class bmiCheckerApplication {
```

Comment:
今さら気づきましたがクラス名は大文字始まりです。
```suggestion
public class BmiCheckerApplication {
```

Reply:
[5980bf6](https://github.com/teddypr/bmi-checker/pull/12/commits/5980bf6dd4447f063728cb49254d78b48487d364) 承知しました。下のコメントのものとまとめてコミットしてしまったのですが、上のパッケージでクラス名を修正しました。